content/en/platform/corda/4.7/enterprise/notary/ha-notary-service-setup.md

```diff
@@ -51,7 +51,7 @@ source has to be monotonic and support leap second smearing.
 * Notary worker configuration files
 
 If you are setting up a local network to test the HA notary setup process, use the [Network Bootstrapper](../network-bootstrapper.md)
-instead of the [Notary Registration Tool](../ha-utilities.md#notary-reg-tool). In all other implementations, the network bootstrapper is not required.
+instead of the [Notary Registration Tool](../ha-utilities.html#notary-reg-tool). In all other implementations, the network bootstrapper is not required.
```

Comment:
```suggestion
instead of the [Notary Registration Tool](../ha-utilities.html#notary-registration). In all other implementations, the network bootstrapper is not required.
```